specification/threatmodel.adoc

```diff
@@ -1,187 +1,300 @@
 [[threatmodel]]
 === Adversary Model
 
-_Unprivileged Software adversary -_ This includes software executing in 
-U-mode managed by S/HS/M-mode system software. This adversary can access 
-U-mode CSRs, process/task memory, CPU registers in the process context 
+_Unprivileged Software adversary -_ This includes software executing in
+U-mode managed by S/HS/M-mode system software. This adversary can access
+U-mode CSRs, process/task memory, CPU registers in the process context
 managed by system software.
```

Comment:
With user space I/O an Unprivileged adversary may also have ability to submit requests to I/O devices.

Reply:
added this vector.